### PR TITLE
CM-397: How to Delete Elements from an Array in C#?

### PR DESCRIPTION
**Pull request by KonstantinosP7**
I committed the branch regarding the CM-397 issue. Please for your review.

### CODE REVIEW

File: collections-arrays/DeleteElementsFromArray/DeleteElementsFromArray.Benchmark/DeleteElementsFromArrayBenchmark.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            return _myArray.Where(e => e != _key).ToArray();
+        }
+        
+        [Benchmark]
+        public int[] DeleteWithLoop() 
+        {
+            var newSize = 0;
+
+            for (int i = 0; i < _myArray.Length; i++) {
+                if (_myArray[i] != _key) {
```
Comment (PhilBroderick) [inline collections-arrays/DeleteElementsFromArray/DeleteElementsFromArray.Benchmark/DeleteElementsFromArrayBenchmark.cs:42]: Again, move `{` onto a new line.

File: collections-arrays/DeleteElementsFromArray/DeleteElementsFromArray.Benchmark/DeleteElementsFromArrayBenchmark.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            }
+
+            Array.Resize(ref _myArray, newSize);
+
+            return _myArray;
+        }
+        
+        private static int[] FillArray() 
+        {
+            Random rnd = new Random();
```
Comment (PhilBroderick) [inline collections-arrays/DeleteElementsFromArray/DeleteElementsFromArray.Benchmark/DeleteElementsFromArrayBenchmark.cs:55]: Can use `var rnd` here.

File: collections-arrays/DeleteElementsFromArray/DeleteElementsFromArray/Methods/DeleteWithFindAllMethod.cs [hunk 3]
```diff
@@ -0,0 +1,7 @@
+﻿namespace DeleteElementsFromAnArray {
+    public static partial class Methods {
```
Comment (PhilBroderick) [inline collections-arrays/DeleteElementsFromArray/DeleteElementsFromArray/Methods/DeleteWithFindAllMethod.cs:2]: Formatting.

File: collections-arrays/DeleteElementsFromArray/DeleteElementsFromArray/Methods/DeleteWithLoopMethod.cs [hunk 4]
```diff
@@ -0,0 +1,18 @@
+﻿namespace DeleteElementsFromAnArray {
+    public static partial class Methods {
```
Comment (PhilBroderick) [inline collections-arrays/DeleteElementsFromArray/DeleteElementsFromArray/Methods/DeleteWithLoopMethod.cs:2]: Likewise.

**Review by PhilBroderick — CHANGES_REQUESTED** [review]
Mostly formatting changes are required.

**Comment by KonstantinosP7** [comment]
Hello Phil, 
I checked through every file of the solution and fixed this formatting issue. Fyi, it was created because I used the shortcut formatting of vs.

**Review by PhilBroderick — CHANGES_REQUESTED** [review]
Just one further change that'll require some changes in the article.

**Review by PhilBroderick — APPROVED** [review]

**Review by PhilBroderick — APPROVED** [review]

**Pull request merged**